### DISCUSSION TITLE
How to automate the installer

### DISCUSSION THREAD

**Discussion by vanderlpp** [netdata/netdata#16879] [Q&A] [answered]
Hi, guys.

I need to automate the installation script

`wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh --stable-channel --claim-token TOKEN --claim-rooms ROOMS --claim-url https://app.netdata.cloud
`
to not ask for "-y" confirmation to increase data on the disk.

Any option to put in this command line so that it is done automatically.

**Comment by ilyam8** [comment]
Hi, @vanderlpp. You can try [--non-interactive](https://learn.netdata.cloud/docs/installing/one-line-installer-for-all-linux-systems#interactivity).

> to not ask for "-y" confirmation to increase data on the disk.

What do you mean "increase data on the disk"? Can you copy/share the output with this question?

**Reply by vanderlpp** [reply]
it asks for confirmation to download 18MB of data to the disk.

I just need this confirmation not to happen.
I'm trying to put it in another script so it can be self-installed by this script.

**Reply by ilyam8** [reply, marked as answer]
> it asks for confirmation to download 18MB

With `--non-interactive` option?

**Reply by vanderlpp** [reply]
`wget -O /tmp/netdata-kickstart.sh https://get.netdata.cloud/kickstart.sh && sh /tmp/netdata-kickstart.sh --non-interactive --stable-channel --claim-token TOKEN --claim-rooms ROOMS --claim-url https://app.netdata.cloud `

this way??

**Reply by tkatsoulas** [reply]
yes
